bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/home/home.html" %}
```

Comment:
Maybe I'm missing some part of the test, but why does this new template inherit from the old home page template? Would this make it hard to remove the old one later?

Reply:
I think because I approached this as a test, but it's not really a test, but it's kind of a test. There's still a fair amount of code shared between the two, and I'm not convinced that this new page is going to be *the* page at this point, so it seemed leaving the inheritance intact was the right move for now.

It wouldn't be very difficult to migrate this to a standalone template if that ends up being our direction.

Reply:
I like your optimism about this new page not lasting long (we've all heard this before), but if it's easy to pick apart still then 👍